pkg/alertmanager/api_grafana.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/prometheus/alertmanager/cluster/clusterpb"
 
+	"github.com/grafana/alerting/definition"
```

Comment:
```suggestion
	"github.com/prometheus/alertmanager/cluster/clusterpb"
	"github.com/grafana/alerting/definition"

```

You need to move this up to the next section to make the linter happy.